### PR TITLE
fix(pci.project.private-network): prevent inputs from overlaping

### PR DESCRIPTION
**Pull request by marie-j**
MANAGER-2672

### CODE REVIEW

**Comment by antleblanc** [comment]
@marie-j There is a typo in  the scope of your commit message (`pic` => `pci`).

File: packages/manager/modules/pci/src/projects/project/private-networks/add/add.html [hunk 1]
```diff
@@ -102,14 +102,16 @@
                                     type="text"
                                     name="{{:: region + '-range-start' }}"
                                     data-ng-model="subnet.start"
-                                    disabled>
+                                    disabled
+                                    style="min-width: 0">
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/private-networks/add/add.html]: Avoiding inline style would be better.

Putting this rule (with a comment) to one of these file would be great:

- manager/packages/manager/modules/pci/src/projects/project/project.less
- manager/packages/manager/modules/pci/src/index.scss

Is that make sense to you?

Comment (marie-j) [reply packages/manager/modules/pci/src/projects/project/private-networks/add/add.html]: This is something specific to the private network creation so it doesn't make sense to have it there

**Comment by antleblanc** [comment]
Even if the ticket is still in `quality check` I will merge this pull request in order to be able to provide a new release branch.

Feel free to correct me it shouldn't handled as is.

Otherwise use label to give more information about ticket status.

Thanks!

**Pull request merged**